### ISSUE TITLE
[BUG] Random text overlapping.

### ISSUE THREAD

**Issue by that-guy-rob** [Fruchtzwerg94/PlantUmlViewer#29]
**Description**
When rendering PlantUML diagrams in Notepad++, the text at seemingly random places in some lines in a multiline definition (example legends or activity diagram definition) wraps incorrectly and overlaps at the beginning of the line. Adding a certain characters like a period or comma seams to fix it but I've noticed that ! or ? doesn't help.  I inserted a special hidden character `&#8203;` which resolves the issue, but this is not an ideal solution.

**Reproduction**
1. Installed the PlantUML plugin in Notepad++.
2. Created a PlantUML diagram with a legend.
3. Observed the text wrapping and overlapping behavior.

**Expected behavior**
The text in the legend should render correctly without overlap or the need for special characters.

Here is some of the code that caused the error:
`### /signup_submit
```plantuml
@startuml
skinparam legendBackgroundColor lightgreen
skinparam defaultFontColor blue
skinparam legendAlignment left
skinparam defaultFontName "Arial Black"
skinparam legendFontSize 12

skinparam activity {
    BackgroundColor lightblue
    BorderColor black
    FontColor darkblue
    FontSize 16
}

legend top
————————————————————————————————
=<b>/signup_submit</b>
————————————————————————————————
  Handles user signup by receiving form data,
  validating email, creating new user,
  and redirecting to homepage.
end legend
start
    :Receive form data;
    :Extract email, username, and password;
    :Validate email format;
    if (Valid email format?) then (yes)
        :Validate username and password;
        if (Valid username and password?) then (yes)
            :Create new user;
            if (Success) then (yes)
                :Set session user;
                :Redirect to homepage;
            else (no)
                :Flash error message;
            endif
        else (no)
            :Flash error message;
        endif
    else (no)
        :Flash error message;
    endif
stop
@enduml

````

This is what happens:
![screenshot_of_error](https://github.com/user-attachments/assets/310fca6e-8a11-4b05-8cc4-b5a418611178)

This is the fix:
````

### /signup_submit
```plantuml
@startuml
skinparam legendBackgroundColor lightgreen
skinparam defaultFontColor blue
skinparam legendAlignment left
skinparam defaultFontName "Arial Black"
skinparam legendFontSize 12

skinparam activity {
    BackgroundColor lightblue
    BorderColor black
    FontColor darkblue
    FontSize 16
}

legend top
————————————————————————————————
=<b>/signup_submit</b>
————————————————————————————————
  Handles user signup by receiving form data,&#8203;
  validating email, creating new user,
  and redirecting to homepage.
end legend
start
    :Receive form data;
    :Extract email, username, and password&#8203;;
    :Validate email format;
    if (Valid email format?) then (yes)
        :Validate username and password;
        if (Valid username and password?) then (yes)
            :Create new user;
            if (Success) then (yes)
                :Set session user;
                :Redirect to homepage;
            else (no)
                :Flash error message;
            endif
        else (no)
            :Flash error message;
        endif
    else (no)
        :Flash error message;
    endif
stop
@enduml

````

This is the result:
![screenshot_of_error2](https://github.com/user-attachments/assets/a97e58be-c0e2-4977-b868-2d75cf4806fb)

**Environment**
 - Windows: 10 Home version 22H2
 - Notepad++ version: [8.6.9 64-bit]
 - PlantUmlViewer version 1.7.0.11

**Additional information**
Add any other information about the problem here.

**Comment by Fruchtzwerg94** [comment]
Hi @that-guy-rob ,

thanks for your report. Please note, the PlantUMLViewer uses the underlying PlantUML library to render the diagrams. In your case, this is a bug of the library which was fixed just recently in the latest version 1.2024.6. Since the currents plugin ships with an older version of the library, it still persists here.
Future releases of the plugin will contain a newer version then. So this issue will be fixed with the next release of PlantUMLViewer.

BR,
Philipp

**Comment by that-guy-rob** [comment]
Awesome! I just wanted to help if I could. Thanks for making it. It's very useful!

________________________________
